src/coreclr/vm/jitinterface.cpp

```diff
@@ -6383,6 +6383,30 @@ CORINFO_CLASS_HANDLE CEEInfo::getTypeInstantiationArgument(CORINFO_CLASS_HANDLE
     return result;
 }
 
+CORINFO_CLASS_HANDLE CEEInfo::getMethodInstantiationArgument(CORINFO_METHOD_HANDLE ftn, unsigned index)
+{
+    CONTRACTL {
+        THROWS;
+        GC_TRIGGERS;
```

Comment:
```suggestion
        NOTHROW;
        GC_NOTRIGGER;
```
`LEAF` JIT/EE interface should be nothrow/notrigger.